SYCL/Basic/multi_device_context.cpp

```diff
@@ -0,0 +1,67 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
+// RUN: %HOST_RUN_PLACEHOLDER %t.out
+// RUN: %CPU_RUN_PLACEHOLDER %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+
+// REQUIRES: cuda
+
+#include <stdlib.h>
+#include <sycl.hpp>
+
+sycl::event add(sycl::queue q, sycl::buffer<int> buff, int *usm,
```

Comment:
nit: Pass by reference instead of pass by value. The tests also serve as documentation/examples, we need to encourage good practices :)

Reply:
done